src/widgets/header/ui/nav-item/nav-item.module.scss

```diff
@@ -49,42 +75,41 @@
     .dropdown-arrow {
       @extend %transition-all;
 
-      margin-left: 10px;
+      cursor: inherit;
+
+      display: flex;
+      align-items: center;
+
+      margin-left: 9px;
```

Comment:
In the project we try not to use margin and also minimize the use of paddings, but this way it will be harder to get into the figma-layout,@ansivgit what do you think?